src/views/landing.vue

```diff
@@ -73,9 +73,11 @@ export default class Landing extends Vue {
 
   async mounted () {
     const { id } = this
-    // if an id was specified then get and load the subject NR
-    if (id) {
-      const nrData = await NamexServices.getNameRequest(true)
```

Comment:
Does this mean we can delete `getNameRequest` from here and from the API now?

Reply:
No, for the anonymous user still need to use getNameRequest function to retrieve NR detail information.

Reply:
So why add the call with the org_id, if it's not needed?

Reply:
For login user, we need the org_id to check the affiliation instead of passing around the session storage values between multiple sites.

Reply:
If we can fetch a NR without the org_id (for anonymous users), why pass the org_id here?

Reply:
How about now?

Reply:
What changed?

My question is, why pass the org_id if you don't have to?

Reply:
The landing page route:

![image](https://user-images.githubusercontent.com/44277752/235005708-895c6a3d-f9e2-4bc7-9e6c-5975142c61ba.png)

if the page with nr id in the route, we expect to see the org id as well. So that we can verify the affiliation with token, org id and nr number together.

Reply:
OK.